### PR TITLE
CIV-5336 first slice of create lip claim diagram

### PR DESCRIPTION
**Pull request by TatsyMaslova**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-5336

### Change description ###

image

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by laarmada — APPROVED** [review]

**Comment by TatsyMaslova** [comment]
reopen because needs testing

**Comment by TatsyMaslova** [comment]
reopen for deploy

**Review by kenneth-hmcts — APPROVED** [review]

File: src/test/java/uk/gov/hmcts/reform/civil/bpmn/CreateClaimSpecTest.java [hunk 1]
```diff
@@ -484,17 +484,6 @@ private void completeCaseAssignment(VariableMap variables) {
         );
     }
 
-    public void startBusinessProcess(VariableMap variables) {
```
Comment (akriti-hmcts) [inline src/test/java/uk/gov/hmcts/reform/civil/bpmn/CreateClaimSpecTest.java:487]: why is this remove?

Comment (kenneth-hmcts) [reply src/test/java/uk/gov/hmcts/reform/civil/bpmn/CreateClaimSpecTest.java]: Adding back for that part. I think it is accidentally removed during development

**Review by akriti-hmcts — APPROVED** [review]

**Pull request merged**